sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md

```diff
@@ -22,64 +22,64 @@ Sets or updates one or more properties' values for a site collection.
- [-LimitedAccessFileType <SPOLimitedAccessFileType>]
- [-AllowEditing <Boolean>]
- [-AnonymousLinkExpirationInDays <Int32>]
- [-OverrideTenantAnonymousLinkExpirationPolicy <Boolean>]
- [-OverrideTenantExternalUserExpirationPolicy <Boolean>]
+ [-ExcludeBlockDownloadPolicySiteOwners <Boolean>]
+ [-ExcludedBlockDownloadGroupIds <Guid>]
  [-ExternalUserExpirationInDays <Int32>]
- [-SensitivityLabel <String>]
- [-RequestFilesLinkExpirationInDays <Int32>]
- [-RequestFilesLinkEnabled <Boolean>]
- [-RemoveLabel]
- [-BlockDownloadPolicy <Boolean>]
- [-OverrideBlockUserInfoVisibility <String>]
- [-OverrideSharingCapability <Boolean>]
- [-DefaultShareLinkScope <SharingScope>]
- [-DefaultShareLinkRole <SharingRole>]
  [-HidePeoplePreviewingFiles <Boolean>]
  [-HidePeopleWhoHaveListsOpen <Boolean>]
- [-AddRestrictedAccessControlGroups <Guid>]
- [-ClearRestrictedAccessControl <Boolean>]
- [-RestrictedAccessControl <Boolean>]
- [-RestrictedAccessControlGroups <Guid>]
- [-RemoveRestrictedAccessControlGroups <Guid>]
- [-ReadOnlyForUnmanagedDevices <Boolean>]
- [-ExcludedBlockDownloadGroupIds <Guid>]
- [-ExcludeBlockDownloadPolicySiteOwners <Boolean>]
- [-ReadOnlyForBlockDownloadPolicy <Boolean>]
- [-AuthenticationContextAccessType <SPOAuthenticationContextPolicyAccessType>]
  [-HubSiteId <Guid>]
  [-InformationBarriersMode <String>]
+ [-LimitedAccessFileType <SPOLimitedAccessFileType>]
+ [-LoopDefaultSharingLinkRole <SharingRole>]
+ [-LoopDefaultSharingLinkScope <SharingScope>]
+ [-OverrideBlockUserInfoVisibility <String>]
+ [-OverrideSharingCapability <Boolean>]
+ [-OverrideTenantAnonymousLinkExpirationPolicy <Boolean>]
+ [-OverrideTenantExternalUserExpirationPolicy <Boolean>]
+ [-ReadOnlyForBlockDownloadPolicy <Boolean>]
+ [-ReadOnlyForUnmanagedDevices <Boolean>]
+ [-RemoveLabel]
+ [-RemoveRestrictedAccessControlGroups [Guid[]]]
+ [-RemoveRestrictedAccessControlGroups <Guid>]
+ [-RequestFilesLinkEnabled <Boolean>]
+ [-RequestFilesLinkExpirationInDays <Int32>]
+ [-ResourceQuotaWarningLevel <Double>]
  [-RestrictContentOrgWideSearch <Boolean>]
  [-RestrictedAccessControl <Boolean>]
- [-RestrictedAccessControlGroups <Guid[]>]
- [-AddRestrictedAccessControlGroups <Guid[]>]
- [-RemoveRestrictedAccessControlGroups <Guid[]>]
- [-ClearRestrictedAccessControl <SwitchParameter>]
+ [-RestrictedAccessControl <Boolean>]
+ [-RestrictedAccessControlGroups [Guid[]]]
+ [-RestrictedAccessControlGroups <Guid>]
+ [-RestrictedToGeo <RestrictedToRegion>]
+ [-SandboxedCodeActivationCapability <SandboxedCodeActivationCapabilities>]
+ [-SensitivityLabel <String>]
+ [-SharingAllowedDomainList <String>]
+ [-SharingBlockedDomainList <String>]
+ [-SharingCapability <SharingCapabilities>] [-StorageQuota <Int64>] [-StorageQuotaWarningLevel <Int64>]
+ [-SharingDomainRestrictionMode <SharingDomainRestrictionModes>]
+ [-ShowPeoplePickerSuggestionsForGuestUsers <Boolean>]
+ [-StorageQuotaReset]
+ [-Title <String>] [-WhatIf] [-AllowDownloadingNonWebViewableFiles <Boolean>]
```

Comment:
Mis-placed [-AllowDownloadingNonWebViewableFiles ...] here and mixed Mutiple params.